development/metamaskbot-build-announce.js

```diff
@@ -39,6 +39,18 @@ async function start() {
       return `<a href="${url}">${platform}</a>`;
     })
     .join(', ');
+  const betaBuildLinks = platforms
```

Comment:
nit: not a blocker for this PR, but maybe we could have a reusable function to spit out the build links per platform for normal, beta, and flask ex `getBuildLinksForPlatform(buildType, platform)`

Reply:
Good suggestion - I'll do this in a follow-up PR so as not to slow down the release